org-roam-capture.el

```diff
@@ -715,7 +715,10 @@ the current value of `point'."
         (when (find-buffer-visiting new-file)
           (kill-buffer (find-buffer-visiting new-file)))
         (delete-file new-file))
-    (org-id-add-location (org-roam-capture--get :id) (org-roam-capture--get :new-file))
+    (when-let ((file (or (org-roam-capture--get :new-file)
```

Comment:
It looks like for even for the `file` options, `:buffer` populated correctly, so it may be best to add remove this line.

Reply:
This is also what I've been experiencing during my brief experiments.